src/ledger/LedgerCloseMetaFrame.cpp

```diff
@@ -20,6 +20,7 @@ LedgerCloseMetaFrame::LedgerCloseMetaFrame(uint32_t protocolVersion)
     if (protocolVersionStartsFrom(protocolVersion, SOROBAN_PROTOCOL_VERSION))
     {
         mVersion = 1;
+        mLedgerCloseMeta.v1().ext.v(1);
```

Comment:
Does this work? You access the v1 extension before setting the version to v1 below.

Reply:
Whoops, fixed.